admin/tests/unit/routes/authenticated/sessions/list/with-required-action-test.js

```diff
@@ -0,0 +1,37 @@
+import Service from '@ember/service';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
+
+module('Unit | Route | authenticated/sessions/list/to-be-published', function(hooks) {
+  setupTest(hooks);
+
+  let store;
+  hooks.beforeEach(function() {
+    class StoreStub extends Service {
+      query = null;
+    }
+    this.owner.register('service:store', StoreStub);
```

Comment:
**question (non bloquante)**
@laura-bergoens à l'occasion je veux bien savoir pourquoi les lignes 11 à 14 sont nécessaires / ou mieux /, que de ne pas les mettre ... on se protège de quoi en disant dans le test que le store est un service ? 
(c'est pas déjà fait par this.owner.lookup('service:store') si je comprends bien ?)